sublime/Toit.sublime-build

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I would rename that to `toit.sublime-build` (lowercase).
It's relatively rare to have capitalized files in Linux.

That said: if you have good reasons for naming it this way just let me know.

Reply:
Turns out that Sublime files often have capitalized files.
Resolving this comment.